resume-buddy/src/main/java/com/google/sps/servlets/ReviewerDataServlet.java

```diff
@@ -51,15 +51,24 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     reviewerEntity.setProperty("first-name", fname);
     reviewerEntity.setProperty("last-name", lname);
     reviewerEntity.setProperty("email", email);
-    reviewerEntity.setProperty("degree", degree);
+    reviewerEntity.setProperty("education-level", degree);
     reviewerEntity.setProperty("school", school);
     reviewerEntity.setProperty("career", career);
     reviewerEntity.setProperty("company", company);
     reviewerEntity.setProperty("years-experience", numYears);
     reviewerEntity.setProperty("submit-date", new Date());
 
+    Entity userEntity = new Entity("User");
+    userEntity.setProperty("first-name", fname);
+    userEntity.setProperty("last-name", lname);
+    userEntity.setProperty("email", email);
+    userEntity.setProperty("degree", degree);
```

Comment:
I think we should consider choosing one of "degree" or "education-level" and using that consistently. That probably includes changing the enum from DEGREE to EDUCATION_LEVEL if that's the direction you'd prefer to go.